mcp-server/DEPLOYMENT.md

```
@@ -0,0 +1,304 @@
+railway up
+```
+
+3. Set environment variables in Railway dashboard:
+```
+ERYXON_API_URL=https://your-project.supabase.co
+UPSTASH_REDIS_REST_URL=...
+UPSTASH_REDIS_REST_TOKEN=...
+```
+
+4. Get your MCP URL:
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)